modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java

```diff
@@ -166,13 +192,34 @@ else if (res.getRemainingTimeToBaselineAdjust() < 0)
         Map<String, VisorBaselineNode> srvs = res.getServers();
 
         // if task runs on a node with VisorBaselineNode of old version (V1) we'll get order=null for all nodes.
+        Function<VisorBaselineNode, String> extractFormattedAddrs = node -> {
+            Stream<String> sortedByIpHosts =
+                Optional.ofNullable(node)
+                    .map(addrs -> node.getAddrs())
+                    .orElse(Collections.emptyList())
+                    .stream()
+                    .sorted(Comparator.comparing(tuple -> new VisorTaskUtils.SortableAddress(tuple.getKey())))
+                .map(addr -> {
+                    if (!addr.getKey().equals(addr.getValue()))
+                        return addr.getValue() + "/" + addr.getKey();
+                    else return addr.getKey();
+                });
+
+            if (verbose) {
+                String hosts = String.join(",", sortedByIpHosts.collect(Collectors.toList()));
+                if (!hosts.isEmpty())
```

Comment:
According to our code style we should add empty line before if here.
Just in case, here are guidelines https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Whitespacesandemptylines